commands/commands_test.go

```diff
@@ -1,11 +1,19 @@
 package commands
 
 import (
+	"io/ioutil"
+	"os"
 	"testing"
 
 	"github.com/github/hub/Godeps/_workspace/src/github.com/bmizerany/assert"
+	"github.com/github/hub/console"
 )
 
+func TestMain(m *testing.M) {
+	console.Default = console.Console{Stdout: ioutil.Discard, Stderr: ioutil.Discard}
```

Comment:
> Since TestMain affects all tests everywhere

This `TestMain` will only affect the commands package. We'd need to put it into a test file in the rest of packages if we want this to be global.

> Also, shouldn't we capture the output instead of discard it?

I'm not aware of any test that asserts the output yet. For [`TestCommandCall`](https://github.com/github/hub/blob/master/commands/commands_test.go#L130) mentioned in #801, we're checking the the function is called and the arguments are correctly passed in.